README.md

```diff
@@ -73,6 +73,14 @@ Rotates the Platform Diffie-Hellman (PDH).
 $ sevctl rotate
 ```
 
+### session
+
+Given a certificate chain file and 32-bit policy, generates base64-encoded GODH and launch session files; as
+well as encoded (not base64) TIK and TEK files.
+
+```console
+$ sevctl session --name {name} --chain {/cert/chain/path} --policy {policy}
```

Comment:
The `--chain` and `--policy` parameters are both mandatory so it feels usual to to be exposing them as parameters, rather than positional arguments. Use of a parameter would normally be for things that are optional. I think it'd make more sense as

```
 $ sevctl session --name {name}  {/cert/chain/path} {policy}
```

Reply:
Fixed